src/benchmark.cc

```diff
@@ -45,8 +45,7 @@ DEFINE_string(benchmark_filter, ".",
               "into the process are run.");
 
 DEFINE_int32(benchmark_iterations, 0,
-             "Total number of iterations per benchmark. 0 means the benchmarks "
-             "are time-based.");
+             "The exact amount of iterations per benchmark");
```

Comment:
Yes. We now just use an iteration count to try and get the benchmarks to run for benchmark_min_time.